acstools/polarization_tools.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
+"""
+Toolkit for analyzing HST/ACS polarization data.
+
+CHANGE LOG
```

Comment:
Hmm. Usually I keep the change log as release notes, but we can start utilizing `CHANGES.rst` for real rather than embedding the change log inside a docstring.